images/cockroachdb/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM BASEIMAGE
+
+ARG ARCH
+ARG TINI_VERSION
+
+# Run tini as PID 1 and avoid signal handling issues
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-${ARCH} /tini
+RUN chmod +x /tini
+
+ADD rook /usr/local/bin/
+
+ENTRYPOINT ["/tini", "--", "/usr/local/bin/rook"]
```

Comment:
I'm not sure we need tini here after looking into it for my review. Do you need Rook to handle signals? If so, which ones? Also, I assume cockroachdb isn't spawning subprocesses, so there's no need for an init process to reap them if cockroachdb crashes.

Reply:
Thanks for your research and legwork on tini @tonya11en.  From the discussion on Slack, it sounds like tini can be removed if:
1) the storage provider will not spawn any child processes
2) we do not need the rook binary as the entry point to provision, setup configuration, or perform tasks such as recovery on behalf of the storage provider daemon.

Based on that list, I think we can start with cockroach not using tini. Thanks!